instance/models/instance.py

```diff
@@ -119,11 +123,13 @@ class Instance(ValidateModelMixin, models.Model):
     # in a query, e.g. to do .select_related('ref_set')
     ref_set = GenericRelation(InstanceReference, content_type_field='instance_type', object_id_field='instance_id')
     openstack_region = models.CharField(
+        # TODO: Move this to OpenedXInstance
         max_length=16,
         blank=False,
         default=default_setting('OPENSTACK_REGION'),
     )
     tags = models.ManyToManyField(
+        # TODO: Move this to InstanceReference (common fields should be in non-abstract tables where possible)
```

Comment:
Why?  I thought about whether to put this on `InstanceReference` or `Instance`, and I couldn't find strong reasons either way.  In the end I chose the way that requires less typing and indirection in the Django shell.